streams/src/test/java/org/apache/kafka/streams/TopologyTest.java

```diff
@@ -252,7 +251,7 @@ public void shouldNotAllowToAddStoreWithSameName() {
         } catch (final TopologyException expected) { }
     }
 
-    @Test(expected = TopologyBuilderException.class)
+    @Test(expected = StreamsException.class)
```

Comment:
I think, we should remove the expected class here (note, that a `StreamsException` might be thrown not just from the last statement that we want to test -- thus, using `expected` the test might pass even if the exception is thrown from a different place what would be an actual failure